modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/service/impl/SXPBlueprintLocalServiceImpl.java

```diff
@@ -77,6 +78,10 @@ public SXPBlueprint addSXPBlueprint(
 		sxpBlueprint.setStatusByUserId(user.getUserId());
 		sxpBlueprint.setStatusDate(serviceContext.getModifiedDate(null));
 
+		sxpBlueprint =
+			_upgradeSXPBlueprintContributorStorageSchemaHelper.
+				upgradeContributorStorageSchema(sxpBlueprint);
+
```

Comment:
it might help clarify whats happening here if we put this logic in a method called something like upgradeImportedBlueprint(). otherwise the "upgrade" terminology feels a little awkward here since people might assume that addSXPBlueprint is creating a brand new blueprint instead of importing one